mlflow/pytorch/__init__.py

```diff
@@ -287,6 +288,46 @@ def gen_data():
     )
 
 
+@experimental
+def log_state_dict(state_dict, artifact_path, **kwargs):
+    """
+    Log a state dict as an MLflow artifact for the current run.
+
+    :param state_dict: PyTorch state dict to be saved.
+    :param artifact_path: Run-relative artifact path.
+    :param kwargs: kwargs to pass to ``torch.save`` method.
+
+    .. code-block:: python
+        :caption: Example
+
+        with mlflow.start_run():
+            state_dict = model.state_dict()
+            mlflow.pytorch.log_state_dict(state_dict, artifact_path="state_dict")
+    """
+
+    with TempDir() as tmp:
+        local_path = tmp.path("model")
+        save_state_dict(state_dict=state_dict, path=local_path, **kwargs)
+        mlflow.tracking.fluent.log_artifacts(local_path, artifact_path)
```

Comment:
```suggestion
        save_state_dict(state_dict=state_dict, path=tmp.path(), **kwargs)
        mlflow.log_artifacts(local_path, artifact_path)
```

The code here can be simplified like this.

Reply:
I have replaced `mlflow.tracking.fluent.log_artifacts` with `mlflow.log_artifacts`. `local_path` variable is assigned as it is used twice, once in `save_state_dict` and other in `mlflow.log_artifacts`. So, can we keep the variable ?

Reply:
Yes. I just wanted to say we don't need `.path("model")`.

Reply:
Yes, we can keep it.

```python
        local_path = tmp.path()
        save_state_dict(state_dict=state_dict, path=local_path, **kwargs)
        mlflow.log_artifacts(local_path, artifact_path)
```